apps/DisplayCluster/resources/MasterContentWindow.qml

```diff
@@ -27,28 +26,36 @@ BaseContentWindow {
             displaygroup.focus(contentwindow.id)
     }
 
-    TouchMouseArea {
+    MultiTouchGestureArea
+    {
         id: windowMoveAndResizeArea
 
-        visible: !contentInteractionArea.visible
-        anchors.fill: parent
+        onTapStarted: displaygroup.moveContentWindowToFront(contentwindow.id)
+
+        onTapEnded: {
+            contentwindow.state = ContentWindow.NONE
+            toggleControlsVisibility()
+        }
 
-        onDoubleTap: toggleFocusMode()
-        onTouchBegin: displaygroup.moveContentWindowToFront(contentwindow.id)
-        onTouchEnd: contentwindow.state = ContentWindow.NONE
-        onTap: toggleControlsVisibility()
         onPan: {
             contentwindow.state = ContentWindow.MOVING
-            contentwindow.controller.moveTo(Qt.point(contentwindow.x + delta.x,
-                                                     contentwindow.y + delta.y))
+            contentwindow.controller.moveTo(Qt.point(contentwindow.x + dx,
+                                                     contentwindow.y + dy))
         }
-        onPinch: {
-            contentwindow.state = ContentWindow.RESIZING
-            contentwindow.controller.scale(position, pixelDelta)
+
+        onPanEnded: contentwindow.state = ContentWindow.NONE
+
+        onDoubleTap: toggleFocusMode()
+
+        PinchWheelArea{
+            onPinchChanged: {
```

Comment:
I cannot name the signal "pinch()". There is a name conflict with the pinch property inside the pincharea.